ids/mapping.json

```diff
@@ -30,5 +30,6 @@
   "swaraj": "swarajpure",
   "sweta": "karnsweta92",
   "taksh": "takshch",
-  "tejas": "tejasberi"
+  "tejas": "tejasberi",
+  "uttam": "Uttam"
```

Comment:
The key is correct, the value should be your github id

Reply:
kachwalUttamsharma
This is my github id. updating it.